variables.tf

```diff
@@ -1,71 +1,104 @@
 ##########################################
-# Common
+# Naming and Tagging
 ##########################################
 
+variable "resource_naming_prefix" {
+  description = "Naming prefix to be applied to all resources created by this module unless explicitly overriden."
+  type        = string
+}
 variable "tags" {
   description = "Tags applied to your resources"
   default     = {}
 }
 
-variable "region" {
-  description = "AWS deployment region"
-  type        = string
-}
-
-variable "account_id" {
-  description = "AWS Account ID"
-  type        = string
-}
-
 ##########################################
 # Sagemaker
 ##########################################
 
 variable "model_name" {
-  description = "Name of the Sagemaker model"
+  description = "Name of the Sagemaker model. If not specified, a default name will be generated using naming prefix."
```

Comment:
I agree, the reason I'm defaulting to "" if because if they don't specify a model name, it just leans on the `resource_name_prefix` for naming. However, I'm tempted to just delete these variables and use `resource_name_prefix` for everything - it's a lot clearer imo.

Reply:
**Done**